Q_hub/static/style.css

```diff
@@ -0,0 +1,83 @@
+nav{
```

Comment:
Sir I looked up for it but couldn't find anything. How can I exclude only the frontend files while making a PR? Does a PR not mean that the VCS compares the master and remote branch and integrates the latest changes of the remote branch into the master. How do frontend files affect the merging process?